build.xml

```diff
@@ -2103,6 +2103,9 @@
           <arg value="${build.test.dir}/jmh-result.json"/>
           <arg value="-v"/>
           <arg value="EXTRA"/>
+          <jvmarg line="${java11-jvmargs}"/>
+          <arg value="-prof"/>
+          <arg value="async:libPath=/usr/local/lib/libasyncProfiler.dylib"/>
```

Comment:
let's split the jmh changes into a separate PR, I'm not sure this is something we want to change for everyone but I don't want to block merging the actual vector changes

Reply:
I understand and agree. So, I removed them [here](https://github.com/datastax/cassandra/pull/690/commits/e39149e6c9ed91921c5bb48ae62c2d877ddc6a9d).